### DISCUSSION TITLE
Selenium web driver version

### DISCUSSION THREAD

**Discussion by tp1845** [community/community#68188] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I want to make python script that work on ubuntu with firefox web browser,
Which selenium web driver works for this?

Thanks in advance

**Comment by netprince1210venus** [comment]
https://github.com/SeleniumHQ/selenium

you can check with this link.

Thanks

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
